composer/trainer/checkpoint.py

```diff
@@ -166,9 +165,9 @@ def _download_checkpoint(self, node_checkpoint_folder: str) -> Tuple[str, Option
                 :meth:`deepspeed.DeepSpeedEngine.load_checkpoint`.
         """
         checkpoint_archive_name = self.path.split(os.path.sep)[-1]
-        rank_zero_checkpoint_archive_name = "rank_0." + checkpoint_archive_name.format(rank=0)
+        rank_zero_checkpoint_archive_name = "rank_0." + checkpoint_archive_name.format(RANK=0)
```

Comment:
Nit: The formatting logic for checkpoint names is specific enough that it ought to be in its own function.

Reply:
Good point, refactored it. Thanks!